src/udp_capture.cpp

```diff
@@ -718,6 +787,7 @@ BFstatus bfUdpCaptureCreate(BFudpcapture* obj,
 }
 BFstatus bfUdpCaptureDestroy(BFudpcapture obj) {
 	BF_ASSERT(obj, BF_STATUS_INVALID_HANDLE);
+	obj->close_logs();
```

Comment:
I also think there is some room for improvement in the logging since the UDP transmit stuff ends up in a separate /dev/shm/bifrost_transmit directory.  It would be cleaned if they could all be in the same /dev/shm/biforst/<PID> directory.  However, that does require being more careful about cleaning up at the end.